src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,101 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_SYNCTHINGSERVIVE = "syncthingservice";
```

Comment:
I'd argue that "SyncthingService" is not really an action. It should at the very least contain a verb, especially if it's a public API.
It also has a typo in the name. Furthermore, it'd be wise to follow common conventions such as keeping the value namespaced, e.g. "android.intent.action.VIEW", so syncthing's actions should be "com.nutomic.syncthingandroid.action.<ACTION_NAME>"